### PR TITLE
Scrivito dark

### PR DESCRIPTION
**Pull request by agessler**
@apepper small fixes for https://github.com/Scrivito/scrivito_example_app_js/pull/428#discussion_r742330193

### CODE REVIEW

File: src/assets/stylesheets/_social_cards.scss [hunk 1]
```diff
@@ -221,3 +230,32 @@ $blueDarken: #0085ac;
...
+    background: rgba(222, 222, 222, 0.2);
+    color: #eee;
+  }
+
+  [data-scrivito-editors-placeholder]:empty:not(:focus):before { color: #aaa!important;}
+
+  .seo_card_preview .card { background: rgba(222, 222, 222, 0.2); }
+  .seo_card_preview .card_text h5 { color: #ccc!important;}
+  .seo_card_preview .card_text p { color:#ddd!important;}
+  .seo_card_preview .creator { color: #5e86be;}
```
Comment (dcsaszar) [inline src/assets/stylesheets/_social_cards.scss:260]: * [ ] FYI This color fails the contrast check https://webaim.org/resources/contrastchecker/?fcolor=5E86BE&bcolor=3F3F3F. In other places (e.g. changed attributes labels) we use a different blue (`#698cbe`), which passes at least for large text https://webaim.org/resources/contrastchecker/?fcolor=698cbe&bcolor=3F3F3F

File: src/assets/stylesheets/_social_cards.scss [hunk 2]
```diff
@@ -40,11 +40,9 @@
   margin-bottom: 5px;
   padding: 0 8px;
   min-height: 30px;
+  line-height: 30px;
 }
-[data-scrivito-display-mode="editing"]
-  [data-scrivito-editors-placeholder]:empty:not(:focus)::before {
-  line-height: 18px !important;
-}
+.seo_card_input .input:focus { box-shadow: 0 0 0 2px #577fb6 inset; }
```
Comment (dcsaszar) [inline src/assets/stylesheets/_social_cards.scss:45]: * [ ] We have two borders around focused inputs, which looks weird and is different from other UI elements:

|  image  | image  |
| -- | -- |
| image  |  image  |

* [ ] FYI Not caused by the current changes, but I noticed it here: The text input color for an input with a warning is not grey, as it is for regular UI inputs with warnings.

**Review by dcsaszar — CHANGES_REQUESTED** [review]
Looks mostly good. I'd like to see at least the double border situation improved.

**Comment by agessler** [comment]
@dcsaszar Regarding last comment. I adapted the focus state and notice appearance - now the social tab looks same as built-in.
 focus

**Comment by agessler** [comment]
@dcsaszar last commit fixes this issue we talked about.

**Review by dcsaszar — APPROVED** [review]
👍

**Pull request merged**